libs/sdk/src/lib/kinetic-sdk-internal.ts

```diff
@@ -220,6 +225,11 @@ export class KineticSdkInternal {
     if (destinations?.length > 15) {
       throw new Error('Maximum number of destinations exceeded')
     }
+    destinations.map((transfer) => {
```

Comment:
We should use `.forEach()` here as the method `map()` is not returning anything.